src/mono/wasm/runtime/library_mono.ts

```diff
@@ -581,8 +540,8 @@ var MonoSupportLib = {
 		mono_wasm_send_dbg_command_with_parms: function (id, command_set, command, command_parameters, length, valtype, newvalue)
```

Comment:
id, command_set, command are numbers.
command_parameters is a base64 string.
length and valtype are numbers.
newvalue is a string.